website/src/server.mjs

```diff
@@ -50,6 +51,9 @@ export function makeServer({
             path.join(router.wwwRootPath, classifiedDirectory.path),
             { currentURI: request.path }
           );
+          const $ = cheerio.load(out);
```

Comment:
Must fix: This code only runs for the development server. It does not run when building the website for production (`yarn build`), but it should.